src/controller/organization.controller.ts

```diff
@@ -169,14 +170,15 @@ export class OrganizationController {
   }
 
   @UseGuards(JwtAuthGuard)
-  @Post(':id/invite/anwser')
+  @Post(':id/invite/answer')
   async inviteResponse(
     @Req() req,
     @Param('id') id: string,
     @Body() response: boolean,
     @Res() res,
   ) {
     const invitationResult: OrganizationInvitation = await this.organizationService.answerInvitation(id, req.user._id, response);
+
     if (invitationResult.status == 'Accepted' && response == true) return res.json({ message: 'User accepted the invitation' });
     if (invitationResult.status == 'Declined' && response == false) return res.json({ message: 'User declined the invitation' });
```

Comment:
you do not need to do these checks, just return `return res.json({ message: 'User answered the invitation' });`
if there's an error, from your service code, it will be thrown as an error.